Project.toml

```diff
@@ -1,17 +1,19 @@
 name = "DustExtinction"
 uuid = "fb44c06c-c62f-5397-83f5-69249e0a3c8e"
 license = "MIT"
-version = "0.6.0"
+version = "0.7.0"
```

Comment:
I may have already asked this in the past, but how about going to v1.0?

Reply:
I think with this change we should be pretty well setup to facilitate any new laws, but let’s sit on the interface a little bit longer before committing. At 1.0 I assume we would drop all the depredations we’ve been holding. I wish I were using this more actively so I could find the pain points.